Source/Rows/Common/FieldRow.swift

```diff
@@ -86,8 +86,25 @@ open class FieldRow<Cell: CellType>: FormatteableRow<Cell>, FieldRowConformance,
     open var keyboardReturnType: KeyboardReturnTypeConfiguration?
 
     /// The percentage of the cell that should be occupied by the textField
-    open var textFieldPercentage: CGFloat?
-
+	@available (*, deprecated, message: "Use titleLabelPercentage instead")
+	open var textFieldPercentage : CGFloat? {
+		get {
+			guard let titlePercentage = titlePercentage else { return nil }
+			return 1 - titlePercentage
+		}
+		set {
+			if let newValue = newValue {
+				titlePercentage = 1 - newValue
+			}
+			else {
+				titlePercentage = nil
+			}
```

Comment:
Similar to [this comment](https://github.com/xmartlabs/Eureka/pull/965#discussion_r129782699).

```
titlePercentage = newValue != nil ? 1 - newValue! : nil
```